docs/4.5-view-patterns.md

```
@@ -981,6 +981,88 @@ echo $this->element('turboActiveTabs', [
+**Order Value Guidelines:**
+
+- **1-10**: Plugin tabs (Officers, Authorizations, Awards, etc.)
+- **10-20**: Primary entity tabs (Members, Roles, Notes)
+- **20-30**: Secondary entity tabs (Additional Info, Settings)
+- **30+**: Administrative or rarely used tabs
+- **999**: Default fallback for tabs without explicit order
+
+**Example Ordering:**
+
+For a Member profile page:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)